motr/io_nw_xfer.c

```diff
@@ -2080,6 +2080,7 @@ static int nw_xfer_req_dispatch(struct nw_xfer_request *xfer)
 	struct m0_op           *op;
 	struct target_ioreq    *ti;
 	struct m0_client       *instance;
+	int                     bulk_error = 0;
```

Comment:
Rename bulk_error to non_rpc_post_error.

Reply:
done